src/telemetry/graph.jsx

```diff
@@ -27,12 +26,12 @@ class TelemetryContainer extends Component {
     return null;
   }
 
-  async componentDidMount() {
-    const { id, queryParams } = this.props;
-    await this.fetchPlotGraph(id, queryParams);
+  componentDidMount() {
+    this.fetchPlotGraph();
```

Comment:
@the6thm0nth   Please keep this async:  
```
  async componentDidMount() {
    return this.fetchPlotGraph();
  }
```

or

```
  async componentDidMount() {
    await this.fetchPlotGraph();
  }
```

This project has higher level components that wrap `componentDidMount()` in a try/catch, just in case anything goes wrong.  Calling an async function like a function breaks this error handing

When you call an async function like a function (without await), you are spawning a new promise chain. Promise chains are like threads; they are sequences of control that run independently from the main flow of control.  You will and will loose any return values, or thrown exceptions from it; your code is now responsible for communicating between the main flow of control and the new promise chain.  Spawning new promise chains is powerful, so there are many good times to use them.  Here is the pattern:

    // define an async function
    const myChain = sync () => {
       /*code for a new promise chain*/
    }
    // run it now
    (myChain)();

sometimes written all together,  to save ourselves naming things:

    (async ()=>{ 
        /*code for a new promise chain*/  
    })(); 

So, whenever you see an async function called as a regular function, realize you are loosing some control.

Reply:
Oh, sorry for that, I didn't know this is used by higher level components. I fixed it.